.github/workflows/build.yml

```diff
@@ -1,5 +1,7 @@
 name: Build
 on:
+  release:
```

Comment:
Please remove this `release` part. Imo it is not necessary since publishing happens automatically now and if something goes wrong we have the manual `workflow_dispatch` if needed. 

Imo triggering it also on release creation will trigger to much publishing/build actions.

Reply:
makes sense, done!